### ISSUE TITLE
Format Version to Game Mapping.

### ISSUE THREAD

**Issue by neomonkeus** [niftools/blender_niftools_addon#94]
A user should be able to select a Nif Format version, which will in-turn provide a selection of possible supported games types for that version.

The selected "Game" is used to determine code execution, returning "Game" specific types. Currently, the "Game" is selected as the highest version. The code does not check if "Game" in Version

Sample Error: Selecting  20.0.0.4 for export, with .kf animation even though the game_list contains OB, CIV4, both of which support .kf's.

Use case for supporting specific version, rather than forcing highest supported version to be mapped to a particular game is that although we treat the "Game" as being for one physical game, other games may also use the same version information.

**Comment by HENDRIX-ZT2** [comment]
Do we really want this? I don't think so, because we treat 'game' as first class citizen on export. Most version checks really check for supported games.

So if we force the user to first select a desired version and then a game from the filtered list of games that use this version, we make access to the game var more difficult than needed.

On the other hand, we could turn the 'version' field into a dropdown that is updated and filtered when a game is selected, to let users chose which particular nif version they want to export for their game of choice.

**Comment by neomonkeus** [comment]
Yeah that is a nice option, I just find it a bit confusing if you import a nif, the version info get updated, but the game stays the same as it was. 
Now this could be a me issue as for the most part I don't know where half the nifs I am using are coming from.

I do like the filterable option, and looking at the nif.xml update that is something that is probably more accessible when we upgrade pyffi.
